docs/platforms/ios.md

```diff
@@ -0,0 +1,68 @@
+inherit.test-requires = "append"
+test-requires = ["arm64-testing-helper"]
+```
+
+This configuration would:
+
+ * Specify a `test-sources` and `test-requires` for all iOS targets;
+ * Add a `PATH` setting that will be used on physical iOS devices; and
+ * Add `arm64-testing-helper` to the test environment for all ARM64 devices (whether simulator or device).
+
+## iOS version compatibility
+
+iOS builds will honor the `IPHONEOS_DEPLOYMENT_TARGET` environment variable to set the minimum supported API version for generated wheels. This will default to `13.0` if the environment variable isn't set.
+
+## Cross platform builds
+
+iOS builds are *cross platform builds*, as it not possible to run compilers and other build tools "on device". The pre-compiled iOS binaries used to support iOS builds include tooling that can convert any virtual environment into a cross platform virtual environment - that is, an environment that can run binaries on the build machine (macOS), but, if asked, will respond as if it is an iOS machine. This allows pip, build, and other build tools to perform iOS-appropriate behaviour.
+
+## Build frontend support
+
+iOS builds support both the pip and build build frontends. In principle, support for uv with the `build[uv]` frontend should be possible, but uv [doesn't currently have support for cross-platform builds](https://github.com/astral-sh/uv/issues/7957), and [doesn't have support for iOS (or Android) tags](https://github.com/astral-sh/uv/issues/8029).
+
+## Build environment
+
+The environment used to run builds does not inherit the full user environment - in particular, `PATH` is deliberately re-written. This is because UNIX C tooling doesn't do a great job differentiating between "macOS ARM64" and "iOS ARM64" binaries. If (for example) Homebrew is on the path when compilation commands are invoked, it's easy for a macOS version of a library to be linked into the iOS binary, rendering it unusable on iOS. To prevent this, iOS builds always force `PATH` to a "known minimal" path, that includes only the bare system utilities, plus the current user's cargo folder (to facilitate Rust builds).
+
+## Tests
+
+If tests have been configured, the test suite will be executed on the simulator matching the architecture of the build machine - that is, if you're building on an ARM64 macOS machine, the ARM64 wheel will be tested on an ARM64 simulator. It is not possible to use cibuildwheel to test wheels on other simulators, or on physical devices.
+
+The iOS test environment can't support running shell scripts, so the `CIBW_TEST_COMMAND` value must be specified as if it were a command line being passed to `python -m ...`. In addition, the test itself must be run "on device", so the local project directory cannot be used to run tests. Instead, the entire project sources must be copied onto the test device; or the project must use `CIBW_TEST_SOURCES` to specify the minimum subset of files that should be copied to the test environment.
```

Comment:
It's unclear whether this means the cibuildwheel user must *do* something to copy the entire sources (in which case it should explain how), or that the default behavior is to copy the entire sources (in which case it should say so, both here and in the `CIBW_TEST_SOURCES` documentation).

Also, what exactly is "the entire project sources"? The directory containing the pypyroject.toml file?

Reply:
There should also be an explanation of how this relates to the PYTHONPATH used for running `python -m`.

Finally, the `CIBW_TEST_SOURCES` documentation says the "temporary folder will be used as the working directory for running the test suite" – is this also true on iOS?

Reply:
This section has changed somewhat because of the change to make `test-sources` a mandatory setting on iOS; but I've clarified some of the other points you've made here (such as clarifying the "temporary folder" on iOS.